src/components/Common/Module/FieldPicker.vue

```diff
@@ -1,49 +1,126 @@
-          <draggable
-            class="drag-area border"
-            :list.sync="availableFields"
-            :group="group"
+        <div class="available mt-1 w-50">
+          <div
+            class="d-flex align-items-center justify-content-between"
           >
-            <div v-for="field in availableFields"
-                 @dblclick="selectedFields = [...selectedFields, field]"
-                 class="field"
-                 :key="field.name">
-              <span v-if="field.label">{{ field.label }} ({{field.name}})</span>
-              <span v-else>{{field.name}}</span>
-              <span v-if="field.isRequired">*</span>
-              <span class="system float-right" v-if="field.isSystem">{{ $t('selector.systemField') }}</span>
-            </div>
-          </draggable>
+            <label
+              class="ml-1 mb-1 text-primary"
+            >
+              {{ $t('selector.available') }}
+            </label>
+            <b-button
+              :class="[selectAll ? 'visible' : 'invisible']"
+              @click.prevent="selectedFields = [...allFields]"
+              variant="link"
+              class="float-right px-0 text-muted m-0"
+            >
+              {{ $t('selector.selectAll') }}
+            </b-button>
+          </div>
+          <b-list-group
+            vertical
+            class="overflow-auto"
+          >
+            <draggable
+              class="drag-area px-1"
+              :disabled="disabled"
+              :group="group"
+              :value="filterAdavailableFields"
+              :key="filterAdavailableFields.length"
+            >
+              <b-list-group-item
+                v-for="field in filterAdavailableFields"
+                :key="field.name"
+                @dblclick="selectField(field)"
+                class="e-resize mb-3 pl-3 border rounded text-truncate"
+              >
+                <field-item
+                  :field="field"
+                  :selected-fields="selectedFields"
+                  :is-right-column="false"
+                  @select-field="selectField"
+                />
+              </b-list-group-item>
+            </draggable>
+          </b-list-group>
         </div>
-        <div class="selected">
-          <label>{{ $t('selector.selected') }}</label>
-          <b-button @click.prevent="selectedFields = []" variant="link" class="float-right">{{ $t('selector.unselectAll') }}</b-button>
-          <draggable
-            class="drag-area border"
-            v-model="selectedFields"
-            :group="group"
+        <div class="selected mt-1 w-50">
+          <div
+            class="d-flex align-items-center justify-content-between"
+          >
+            <label
+              class="mb-0 ml-1 text-primary"
+            >
+              {{ $t('selector.selected') }}
+            </label>
+            <b-button
+              @click.prevent="selectedFields = []"
```

Comment:
This will be strongly enforced soon, but lets try it now already.
Lets try to respect the prop order of components

1. vue things (v-if, v-for...)
2. static props(a="b")
3. dynamic props (:a="b")
4. classes and styles
5. events (@foo)